_posts/2017-10-05-apprenticeship-at-8thlight.md

```diff
@@ -0,0 +1,213 @@
+
+Another thing I heard is a thing at 8th Light, is juggling. Everyone in the company learns juggling with at least 3 balls. This might seem strange but one thing it surely does to most is get them out of their comfort zone. It also gets you away from your desktop and learn something that simply requires practicing. And so does writing good code.
+
+## Mentors
+
+Every apprentice has a mentor and a co-mentor. The co-mentor supports the mentor and is also there to remove potential biases. I might refer to both just by using “mentor”, since there seems not to be a big differentiation in what a mentor and co-mentor do.
+The mentor is not a teacher who stands beside their apprentice all the time, they are more like guides that help the apprentice to steer their learnings. I heard from apprentices that talk about their mentors as one of the most important people during their apprenticeship.
+Mentoring needs some experience too, the mentor Daniel Irvine writes in [his blog][4-tips-for-mentoring] “go too fast and they won't take everything in; go too slow and they'll be bored”. Mentors must not be too specific, leave room for learning and find the right dose of challenge.
+“Mentors are training their future team members, so they are personally invested in the success of each apprentice” is written [on the 8th Light apprenticeship page][8thlight-apprenticeship].
+
+The time a mentor invests is about half a day a week. There are mentors who take more time to work with their apprentices, but there is no fix rule. 
+
+[8thlight-apprenticeship]: https://8thlight.com/apprenticeship/
+[4-tips-for-mentoring]: https://dev.to/d_ir/4-tips-for-mentoring-developers-894
+
+## Regular Meetings
+
+As mentioned above there are regular meetings between apprentice, mentor and co-mentor. The one and most important meeting is the IPM, the iteration planning meeting. It takes place once a week for about one hour. During this meeting the apprentice reports the progress of the last iteration and asks questions that might come up. The mentors will help answering those questions and might adjust the plan for the upcoming iterations. Later the next iteration is planned. The tasks are moved from the backlog into the to-do column and the apprentice estimates them. The mentors support doing this as well as possible.
+
+All the mentors meet once a month for the MMM, the monthly mentor meeting. In this meeting the mentors exchange ideas on the mentoring and also on the apprentices.
+Especially when in doubt about an apprentice’s progress this meeting serves to support each other.
+
+The review board is another meeting where a selected group of people comes together for reviewing a certain apprentice. This board meets at the end of the apprenticeship to evaluate the apprentice. If there is a veto option by this board or not, I don’t know. The important thing I think is that this board comes together to evaluate a certain apprentice. If this takes place more often I don’t remember.
```

Comment:
We give a set of challenges at the end of the apprenticeship (last two weeks). The review board assesses the challenges as well as the entirety of the apprenticeship to give the recommendation to hire or not. We now select the review board from the start of the apprenticeship, and most apprentices will have at least one mini review board during their apprenticeship where the board will provide feedback on a larger problem that the apprentice has solved.